letsencrypt/client.py

```diff
@@ -378,8 +378,6 @@ def deploy_certificate(self, domains, privkey_path, cert_path, chain_path):
         # sites may have been enabled / final cleanup
         self.installer.restart()
 
-        display_ops.success_installation(domains)
```

Comment:
Ok, I can see it's been moved to `cli.py`. I'm actually very happy about this change, as it allows us to decouple `IDisplay` from the API (`client.py`)